### DISCUSSION TITLE
Education benefits

### DISCUSSION THREAD

**Discussion by wyh-98** [community/community#137130] [GitHub Education]
### Hi everyone,

Since my submission was approved on August 27, I have not received my education allowance

**Comment by Yackens** [comment]
It takes some time, depending on the application date, number of applicants, location... 

If it takes more than a month, my advice is that you reach them through the [GitHub Education Support Portal](https://support.github.com/contact/education), or you might find contact options in the emails they’ve sent you.

If you found this answer useful, don't forget to mark it as the accepted answer so more people can see it.

**Comment by davevad93** [comment]
Hi @wyh-98 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @wyh-98 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
